tawazi/dag.py

```diff
@@ -934,14 +946,12 @@ def setup(self, twz_nodes: Optional[List[Alias]] = None) -> None:
         self.dag.setup(twz_nodes)
 
     def __call__(self, *args: Any) -> Any:
+        if self.executed:
+            raise TawaziUsageError("The DAGExecution is already executed; instantiate a new one")
```

Comment:
wdyt about `DAGExection object should not be reused. Instantiate a new one` ?

Reply:
I like this message more. Fixing it